sdk/identity/identity/README.md

```diff
@@ -1,56 +1,126 @@
 
-## Examples
+## Getting started
 
-### Examples for SDK Users
+### Prerequisites
 
-The `@azure/keyvault-keys` library supports the use of `TokenCredential` for authenticating
-requests to Azure Key Vault.  Here's how you would use the `ClientSecretCredential` to authenticate
-SDK calls:
+- an Azure subscription
+  - if you don't have one, you can sign up for a [free account](https://azure.microsoft.com/free/)
+- Node.js 8 LTS or higher
 
-```javascript
-import { KeysClient } from "@azure/keyvault-keys";
-import { ClientSecretCredential } from "@azure/identity";
+**NOTE:** These credential implementations are not supported in the browser yet.  We will provide browser-ready implementations in a future preview release.
 
-const clientId = process.env["CLIENT_ID"] || "";
-const clientSecret = process.env["CLIENT_SECRET"] || "";
-const tenantId = process.env["TENANT_ID"] || "";
-const vaultName = process.env["KEYVAULT_NAME"] || "<keyvault-name>"
+### Install the package
 
-const url = `https://${vaultName}.vault.azure.net`;
-const credential = new ClientSecretCredential(tenantId, clientId, clientSecret)
-const client = new KeysClient(url, credential);
-const getResult = await client.getKey("MyKeyName");
+Install Azure Identity with `npm`:
+```sh
+npm install --save @azure/identity
 ```
 
-If you use the standard environment variable names `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and
-`AZURE_CLIENT_SECRET` for storing your AAD client credentials, the previous example can be simplified by using `EnvironmentCredential`:
+## Key concepts
+
+### Credentials
+
+Azure Identity offers a variety of credential classes that are accepted by Azure SDK data plane clients. Each client library documents its Azure Identity integration. Azure SDK management
+libraries do not accept these credentials.
+
+Credentials differ mostly in configuration:
+
+|credential class|identity|configuration
+|-|-|-
+|`DefaultAzureCredential`|service principal or managed identity|none for managed identity; [environment variables](#environment-variables) for service principal
+|`ManagedIdentityCredential`|managed identity|none
+|`EnvironmentCredential`|service principal|[environment variables](#environment-variables)
+|`ClientSecretCredential`|service principal|constructor parameters
+|`ClientCertificateCredential`|service principal|constructor parameters
+
+Credentials can be chained and tried in turn until one succeeds; see [chaining credentials](#chaining-credentials) for details.
+
+### DefaultAzureCredential
+`DefaultAzureCredential` is appropriate for most scenarios. It supports authenticating as a service principal or managed identity. To authenticate as a service principal, provide configuration in environment variables as described in the next section.
```

Comment:
It is helpful to know the fallback logic here i.e first we try this and then that.